conan/cli/printers/graph.py

```diff
@@ -105,10 +110,15 @@ def _format_requires(title, reqs_to_print):
             return
         output.info(title, Color.BRIGHT_YELLOW)
         for pref, (status, remote) in sorted(reqs_to_print.items(), key=repr):
-            if remote is not None and status != BINARY_SKIP:
+            orig_status = status
+            if remote is not None and orig_status != BINARY_SKIP:
                 status = "{} ({})".format(status, remote.name)
             name = pref.repr_notime() if status != BINARY_SYSTEM_TOOL else str(pref.ref)
-            output.info("    {} - {}".format(name, status), Color.BRIGHT_CYAN)
+            msg = "    {} - {}".format(name, status)
+            if orig_status == BINARY_SKIP:
+                output.verbose(msg, Color.BRIGHT_CYAN)
```

Comment:
I am missing the summary of skipped binaries

Reply:
Do you want a summary of skipped binaries? Do you mean even if you're not adding `-v` to the command?

Reply:
Yes, in short form, not all the full references, just the pkg/version in a single line or something, but I think it is useful to have it there